docs/guides/snapshot/process.md

```diff
@@ -31,7 +31,7 @@ pod/mypod condition met
 After the pod is running, we can try to get the list of running processes again:
 
 ```bash
-$ kubectl gadget process-collector -n demo
+$ kubectl gadget snapshot process -n demo
```

Comment:
There is one instance missing update on L18.

Reply:
All comments addressed.